src/server_manager/ui_components/outline-server-view.html

```diff
@@ -866,9 +815,6 @@ <h3>[[localize('data-limits-dialog-title')]]</h3>
       // is on the screen (e.g. selected in iron-pages). If help bubbles
       // are initialized before this point, setPosition will not work and
       // they will appear in the top left of the view.
-      showGetConnectedHelpBubble: function() {
```

Comment:
That's a good point. Without this bubble new users might not know how to connect to their server.

Do we want to make the effort to show this bubble the first time someone opens up the share dialog? 
Or alternatively repurpose the bubble to point out that you can connect from the share dialog?

My gut feeling is that a help bubble on a dialog isn't a great user experience.

Reply:
> My gut feeling is that a help bubble on a dialog isn't a great user experience.

Agreed.

> Do we want to make the effort to show this bubble the first time someone opens up the share dialog?

I like this idea, as it creates a distinction between the actions the user may take.

Reply:
Yeah, I see your point that it could be helpful to new users. I went ahead and added it back in and pointed it to the share button. We may want to consider updating the text of the bubble.